WooCommerce/Classes/ViewRelated/InAppPurchases/InAppPurchasesDebugView.swift

```diff
@@ -104,6 +108,10 @@ struct InAppPurchasesDebugView: View {
 
 struct InAppPurchasesDebugView_Previews: PreviewProvider {
     static var previews: some View {
-        InAppPurchasesDebugView(siteID: 0)
+        InAppPurchasesDebugView()
     }
 }
+
+private enum Constants {
+    static let siteIdEnvironmentVariableName = "iap-debug-site-id-purchase-param"
```

Comment:
Nitpicky, but maybe `iap-debug-site-id` is descriptive enough

Reply:
Indeed! That name is simpler and descriptive enough